### PR TITLE
Add weekly highlights for 2022-09-22

### PR DESCRIPTION
**Pull request by marians**
Please add your contributions until September 22, 12:00 CET.

### CODE REVIEW

**Comment by ljakimczuk** [comment]
From Honeybadger's side:

* created a public doc clarifying the concept of app bundle and its place in the App Platform.

File: src/content/changes/highlights/2022-09-22.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+---
+
+## General
+
+## User interfaces
+
+## Apps
+
+## Documentation
+
```
Comment (gianfranco-l) [inline src/content/changes/highlights/2022-09-22.md:17]:
```suggestion
Created a dedicated page clarifying the concept and value of [App Bundle](https://docs.giantswarm.io/app-platform/app-bundle/) and its place in the App Platform.
```

**Comment by gianfranco-l** [comment]
added contribution from honeybadger side

**Comment by weatherhog** [comment]
nothing from atlas or cabbage this week

File: src/content/changes/highlights/2022-09-22.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## User interfaces
+
+As of the latest version, our web UI is ready for use on Google Cloud Platform (GCP) installations.
+
+kubectl-gs [v2.23.0](https://github.com/giantswarm/kubectl-gs/releases/tag/v2.23.0) adds the capability to create a client certificate for workload cluster access with a specific CN prefix. This means you can bind the client presenting this certificate to an RBAC role as a user. Check the `login` command's new `--cn-prefix` flag.
+
```
Comment (stone-z) [inline src/content/changes/highlights/2022-09-22.md:15]:
```suggestion

## Apps

- [security-pack-helper](https://github.com/giantswarm/security-pack-helper) version [v0.0.1](https://github.com/giantswarm/security-pack-helper/blob/main/CHANGELOG.md#001---2022-09-20) initial release supports watching and deleting Kyverno `ReportChangeRequest` resources when they exceed a configured threshold.
- [trivy-app](https://github.com/giantswarm/trivy-app) version [v0.6.0](https://github.com/giantswarm/trivy-app/blob/main/CHANGELOG.md#060---2022-09-16) updates to Trivy version `0.30.4`.

```

Comment (marians) [reply src/content/changes/highlights/2022-09-22.md:15]: Sorry, but the announcement is already sent. We'll have to announce this with next week's round.

**Comment by marians** [comment]
Quick approval, anyone?

**Review by anvddriesch — APPROVED** [review]

**Pull request merged**